projects/core/src/global-message/http-interceptors/handlers/unknown-error/unknown-error.handler.ts

```diff
@@ -32,7 +34,15 @@ export class UnknownErrorHandler extends HttpErrorHandler {
   }
 
   handleError(_request: HttpRequest<any>, errorResponse: HttpErrorResponse) {
-    if (isDevMode() || this.isSsr()) {
+    const shouldLogError = this.featureConfigService.isEnabled(
+      'ssrStrictErrorHandlingForHttpAndNgrx'
+    )
+      ? isDevMode()
+      : isDevMode() || this.isSsr();
+
+    // Error will be handled and logged by the `HttpErrorHandlerInterceptor`.
+    // After removing the `ssrStrictErrorHandlingForHttpAndNgrx` feature flag, error will be logged only in dev mode.
```

Comment:
```suggestion
[NITPICK]
```suggestion
    // Error is already handled and logged by the `HttpErrorHandlerInterceptor`,
    // if `ssrStrictErrorHandlingForHttpAndNgrx` feature toggle is enabled.
    // In the future, after removing the `ssrStrictErrorHandlingForHttpAndNgrx` feature toggle,
    // error will be logged here only in dev mode.
```